### DISCUSSION TITLE
Use rclone to mount the Windows shared folder to the SDCard

### DISCUSSION THREAD

**Discussion by Howard20181** [LSPosed/MagiskOnWSALocal#233] [Ideas]
![image](https://user-images.githubusercontent.com/40033067/198893259-6d6013b6-b35f-4129-88de-b45464520aa0.png)
![image](https://user-images.githubusercontent.com/40033067/198893290-7af34f0b-9ed1-4981-96b5-fe7ddb310e88.png)

Now we can use the [rclone Magisk module (forked by me)](https://github.com/Howard20181/rclone-mount-magisk/tree/fix) to mount the Windows shared folder to the WSA's built-in storage, which facilitates the data transmission from Windows to WSA.

The packaged module can be download from [here](https://github.com/Howard20181/rclone-mount-magisk/releases/latest) (Of course, you can also pack it yourself.).

My branch has made some repairs and adjustments for WSA. Changes will submit to [upstream](https://github.com/AvinashReddy3108/rclone-mount-magisk) after all the code has been checked. Before it is ready, I will keep the branches separated.

**Comment by williamvyong** [comment]
do u have the step how to do it for the noobies ?

**Reply by Howard20181** [reply]
1. Install module in Magisk
1. Reboot
1. Connect to WSA using adb shell
1. Switch user to root
1. Type `rclone config`

**Comment by williamvyong** [comment]
which should i choose ?? local disk ?

**Reply by Howard20181** [reply]
I don't know what you said. Where you make a choose?

**Comment by NiffirgkcaJ** [comment]
Do you have any guides that will make this easy for newbies?
